pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -443,6 +443,7 @@ type Input struct {
 
 // BEGIN HostDevice -----------------------------
 type HostDevice struct {
+	XMLName   xml.Name         `xml:"hostdev"`
```

Comment:
I noticed that [Devices populates []HostDevies](https://github.com/kubevirt/kubevirt/pull/4741/commits/4529ef4e309703273719909223a22d2ad7c39235#diff-66ca092e3693da97e42a13a63ea9d298df19057a358558314e23097bc32c3035R383) and already has `hostdev` annotation.
Doesn't it contradict with Devices annotation?
Will there be two `hostdev` tags?

Reply:
So there will be no tags duplication's?

Reply:
I do not even know what that means really.
The priority specifies from what source to use it, which means only one can exist (otherwise, why a priority is needed?).

This is also used in other places where decoding is needed on a specific sub-tree like in this case.

If you see a problem, please show/demonstrate it so it will be easier to understand what is wrong here.